### ISSUE TITLE
[2021.03.16-17] 오늘의 페이퍼

### ISSUE THREAD

**Issue by unizard** [sejong-rcv/AwesomeMagazine#14]
### 오늘의 페이퍼 평
- 부쩍 Lidar Segmentation 페이퍼들이 많아져 보인다. 갑자기 왜? 이런 경향성이 보이는 걸까?
- Radar Fusion 한번 쯤은 검토해 보고 싶었다.
- "Refine Myself by Teaching Myself" 토픽을 생각만 하고 있었는데, 페이퍼로 보이니 신기하네?

**Comment by unizard** [comment]
- [Student-Teacher Learning from Clean Inputs to Noisy Inputs](https://arxiv.org/pdf/2103.07600.pdf), CVPR2021, 코드X, 이론페이퍼
  - Knowledge Distillation 을 어떻게 하면 잘할 것인가가 핵심
    - [컨트리뷰션 요약] 
    - The student should not be trained to zero training loss.
    - A knowledgeable teacher is generally preferred, but there are limitations.
    - Well-decomposed knowledge leads to better knowledge transfer.

**Comment by unizard** [comment]
- [Refine Myself by Teaching Myself :Feature Refinement via Self-Knowledge Distillation](https://arxiv.org/pdf/2103.08273.pdf), CVPR2021, [코드](https://github.com/MingiJi/FRSKD),  Pytorch

**Comment by unizard** [comment]
### Point Cloud Segmentation
- [LRGNet: Learnable Region Growing for Class-Agnostic Point Cloud Segmentation](https://arxiv.org/pdf/2103.09160.pdf)
- [Lite-HDSeg: LiDAR Semantic Segmentation Using Lite Harmonic Dense Convolutions](https://arxiv.org/pdf/2103.08852.pdf)
- [S3Net: 3D LiDAR Sparse Semantic Segmentation Network](https://arxiv.org/pdf/2103.08745.pdf)

**Comment by unizard** [comment]
###  Radar Fusion
- [Radar Camera Fusion via Representation Learning in Autonomous Driving](https://arxiv.org/pdf/2103.07825.pdf)

**Comment by unizard** [comment]
### 4K Stereo (자율주행혁신기술개발)
- [ORStereo: Occlusion-Aware Recurrent Stereo Matching for 4K-Resolution Images](https://arxiv.org/pdf/2103.07798.pdf)

**Comment by unizard** [comment]
### 딥러닝기반 카메라포즈 추정
- [Back to the Feature: Learning Robust Camera Localization from Pixels to Pose](https://arxiv.org/pdf/2103.09213.pdf), CVPR2021
  - [코드](https://github.com/cvg/pixloc), 공개예정

**Comment by unizard** [comment]
### 학습기반 지역불변특징량
- [RoRD: Rotation-Robust Descriptors and Orthographic Views for Local Feature Matching], (https://arxiv.org/pdf/2103.08573.pdf), ICRA2021, QUT 
  - [웹사이트](https://uditsinghparihar.github.io/RoRD/) , [동영상](https://www.youtube.com/watch?v=4n6_6TMnlOc)
  - <img width="893" alt="스크린샷 2021-03-17 오후 7 54 04" src="https://user-images.githubusercontent.com/11758940/111456593-88641980-875a-11eb-9f1b-0eabd40d1081.png">

**Comment by unizard** [comment]
### 윤성로교수님연구실 & 관심사
- [BBAM: Bounding Box Attribution Map for Weakly Supervised Semantic and Instance Segmentation](https://arxiv.org/pdf/2103.08907.pdf), CVPR2021
  - [코드](https://github.com/jbeomlee93/BBAM), 공개예정
